pkgs/development/python-modules/fmpy/default.nix

```diff
@@ -0,0 +1,153 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, cmake
+, attrs
+, lark-parser
+, lxml
+, msgpack
+, numpy
+, scipy
+, pytz
+, dask
+, requests
+, matplotlib
+, pyqt5
+, pyqtgraph
+, notebook
+, plotly
+}:
+let
+  # Auxiliary library, maybe not extremely useful by itself.
+  rpclib = stdenv.mkDerivation {
+    pname = "rpclib";
+    version = "2.3.0";
+
+    src = fetchFromGitHub {
+      owner = "rpclib";
+      repo = "rpclib";
+      rev = "v2.3.0";
+      sha256 = "0dlbkl47zd2fkxwbn93w51wmvfr8ssp4zribn5wi4cpiky44a4g9";
+    };
+
+    nativeBuildInputs = [ cmake ];
+  };
+
+  cvode = stdenv.mkDerivation {
+    pname = "cvode";
+    version = "5.3.0";
+
+    src = fetchFromGitHub {
+      owner = "LLNL";
+      repo = "sundials";
+      rev = "v5.3.0";
+      sha256 = "08q9svjdl6fmqrwyd9p12m98n0wd2lws38yh223wyp6590zi3i49";
+    };
+
+    nativeBuildInputs = [ cmake ];
+
+    cmakeFlags = [
+      "-DBUILD_ARKODE=OFF"
+      "-DBUILD_CVODES=OFF"
+      "-DBUILD_IDA=OFF"
+      "-DBUILD_IDAS=OFF"
+      "-DBUILD_KINSOL=OFF"
+      "-DEXAMPLES_ENABLE_C=OFF"
+    ];
+  };
+
+in
+
+buildPythonPackage {
+  pname = "FMPy";
+  version = "0.3.1";
+  disabled = pythonOlder "3.5";
+
+  src = fetchFromGitHub {
+    owner = "CATIA-Systems";
+    repo = "FMPy";
+    rev = "v0.3.1";
+    sha256 = "0ig08323id84nd9y272hyccky33my2nlhl44m41fxnkrg6xq7z3m";
+  };
+
+  nativeBuildInputs = [ cmake pyqt5 ];
+
+  propagatedBuildInputs = [
+    cmake
```

Comment:
You are sure about this?

Reply:
Double checked. You are right, I'm not sure. The FMPy generates `CMakeLists.txt`, but it runs `cmake` by itself only in tests, not in regular case.